Library/Homebrew/formula.rb

```diff
@@ -1615,6 +1615,10 @@ def shared_library(name, version = nil)
   # </pre>
   sig { params(source: Pathname, target: Pathname).returns(String) }
   def rpath(source: bin, target: lib)
+    unless target.to_s.start_with?(HOMEBREW_PREFIX)
+      raise "`target` should only be used for paths inside HOMEBREW_PREFIX!"
```

Comment:
```suggestion
      raise "rpath `target` should only be used for paths inside HOMEBREW_PREFIX!"
```

should make it clearer it's referring to the rpath functional call